src/mlpack/methods/reinforcement_learning/environment/continuous_double_pole_cart.hpp

```diff
@@ -0,0 +1,374 @@
+/**
+ * @file continuous_double_pole_cart.hpp
+ * @author Rahul Ganesh Prabhu
+ *
+ * This file is an implementation of Continuous Double Pole Cart Balancing
+ * Task.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_RL_ENVIRONMENT_CONTINUOUS_DOUBLE_POLE_CART_HPP
+#define MLPACK_METHODS_RL_ENVIRONMENT_CONTINUOUS_DOUBLE_POLE_CART_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace rl {
+
+/**
+ * Implementation of Continuous Double Pole Cart Balancing task.
```

Comment:
Can we add some more information about the environment, reference the existing Cart Pole env, and say that this is an extend could be a solution. What do you think?